pkg/config/polling_manager.go

```diff
@@ -157,6 +158,7 @@ func (cm *PollingProjectConfigManager) SyncConfig(datafile []byte) {
 	}
 	cmLogger.Debug(fmt.Sprintf("New datafile set with revision: %s. Old revision: %s", projectConfig.GetRevision(), previousRevision))
 	cm.projectConfig = projectConfig
+	cm.optimizelyConfig = NewOptimizelyConfig(projectConfig)
```

Comment:
We should do this lazily. So if `cm.optimizelyConfig != nil` then we create the optimizelyConfig

Reply:
I will address that on the optimizely config